libavformat/movenc.c

```diff
@@ -4447,8 +4451,9 @@ static int mov_write_traf_tag(AVIOContext *pb, MOVMuxContext *mov,
         }
     }
 
-    if (track->cenc.aes_ctr) {
-        ff_mov_cenc_write_stbl_atoms(&track->cenc, pb, moof_offset);
+    if (track->cenc.encryption_scheme != MOV_ENC_NONE) {
+        ff_mov_cenc_write_senc_tag(&track->cenc, pb, moof_offset);
```

Comment:
With these new changes, is original cenc (full sample encryption) still working?

Reply:
Yes